.github/workflows/test.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         run: ./gradlew classes
       - name: Assemble
         run: ./gradlew assemble
+      - name: Validate Docker Configurations
+        run: find . -name docker-compose\*.yml | while read -r file; do echo "$file"; docker compose -f "$file" config -q; done
```

Comment:
Could you use `docker-compose ... && docker compose ...` please? Obviously we have one case where V2 is more permissive than V1, but for all we know there could be other cases where V1 is more permissive, and I think we should catch both.

Reply:
Hmm, is one form really more permissive than the other? The only difference I see is in the error message being reported, not an absence of error reporting from either tool.

Reply:
Ohh, that's right, thanks! I'll add that.